src/components/10-atoms/checkbox/icon/tick-hard-edges.svg.js

```diff
@@ -0,0 +1 @@
+export default '<svg width="14px" height="10px" viewBox="0 0 14 10" fill="qurrentColor" xmlns="http://www.w3.org/2000/svg">    <path d="M4.57382 7.88591L1.41384 4.77629L0.333984 5.82774L4.57382 10L13.6673 1.05145L12.5988 0L4.57382 7.88591Z" fill="qurrentColor" /></svg>';
```

Comment:
- Remove empty spaces between svg and path tags.

- Typo here: fill="**q**urrentColor"

Reply:
can't see any typo...🙃

Reply:
currentColor .... ? i just can see it there ...

Reply:
Ah I see :)